bdd/boot-vault/jx-requirements.yml

```diff
@@ -14,13 +14,15 @@ environments:
   - key: production
     owner: ""
     repository: ""
+gitops: true
 ingress:
-  domain: ""
-  externalDNS: false
+  domain: boot.bdd.jenkins-x.rocks
+  externalDNS: true
+  namespaceSubDomain: -jx.
   tls:
-    email: ""
-    enabled: false
-    production: false
+    email: jenkins-x@googlegroups.com
+    enabled: true
+    production: true
```

Comment:
I'm not sure we should be using the production lets encrypt service during testing, we will likely be rate limited via our domain or email address.  For testing they recommend using the staging service.

Reply:
We cannot really test E2E with certificates issued by staging let's encrypt service. They are self-singed and various clients fail along the way because the certificate is invalid. I have already tried this but it doesn't work.

The rate limit seems to be applied on `*.bdd.jenkins-x.rocks` TLD. I am thinking that we can register 10-15 TLDs (e.g. bdd1.jenkins-x.rocks, bdd2.jenkins-x.rocks...) and rotate through them every time a BDD test is started. What's your opinion on this?

Reply:
Makes sense to me.

Reply:
I registered 12 DNS managed zones and rotate them each hour. Let's see how it works.